src/Calculator.java

```diff
@@ -1,21 +1,92 @@
+import java.util.Scanner;
+import java.math.*;
+
 public class Calculator {
 
-    // global variables
-    private static final Scanner scanner = new Scanner(System.in);
+	// global variables
+	private static final Scanner scanner = new Scanner(System.in);
+
+	/*
+	 * your code comes here
+	 */
+
+	public static void main(String[] args) {
+		float a = 0;
+		float b = 0;
+		int calc = 1;
+		while (calc == 1) {
+		
+		
+		System.out.println("Bitte Eingabe w�hlen, f�r Kommazahl bitte 1 w�hlen, sonst weiter mit beliebiger Taste: ");
+		int eing = scanner.nextInt();
+		if (eing == 1) {			
+			System.out.println("Bitte ersten Wert eingeben: ");
+			a = scanner.nextFloat();
+
+			System.out.println("Bitte zweiten Wert eingeben; ");
+			b = scanner.nextFloat();
+		} else {
+			System.out.println("Bitte ersten Wert eingeben: ");
+			a = scanner.nextInt();
```

Comment:
For integers, you should also use the data type `int` and not `float`. Therefore you will need 4 different variables.